programs/eosio-launcher/main.cpp

```diff
@@ -989,10 +1016,18 @@ launcher_def::write_config_file (tn_node_def &node) {
 
   if(!is_bios) {
      auto &bios_node = network.nodes["bios"];
-     cfg << "p2p-peer-address = " << bios_node.instance->p2p_endpoint<< "\n";
+     if (p2p == p2p_plugin::NET) {
```

Comment:
In keeping with the net_plugin default, shouldn't this be reversed?  Specific to p2p_plugin::BNET, otherwise anything else gets the other setting.

Reply:
IRL discussion occurred and this is okay.